stable/sumologic-fluentd/values.yaml

```diff
@@ -79,4 +104,12 @@ resources:
     cpu: 256m
     memory: 256Mi
 
-rbac: {}
+rbac:
+  ## If true, create and use RBAC resources
+  create: false
+
+  ## Ignored if rbac.create is true
+  serviceAccountName: default
+
+  # RBAC api version (currently either v1beta1 or v1alpha1)
+  apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
Unneeded. We're only supporting K8s 1.6 & 1.7 right now, so the apiVersion can be statically set in the manifests to `v1beta1`.